orienteer-pivottable/src/main/java/org/orienteer/pivottable/PivotTableModule.java

```diff
@@ -19,6 +19,7 @@
 	public static final String NAME = "pivottable";
 	public static final String WIDGET_OCLASS_NAME = "PivotTableWidget";
 	public static final String OPROPERTY_PIVOT_TABLE_CONFIG = "pivotTableConfiguration";
+	public static final String OPROPERTY_PIVOT_CUSTOM_SQL = "pivotCustomSQL";
 	
 	protected PivotTableModule() {
 		super(NAME, 1, OWidgetsModule.NAME);
```

Comment:
You are modifing datamodel: version should be increased to allow merge changes on already existing databases.